content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -277,74 +281,92 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
                 </h1>
               </div>
               <span>
-                <NextLink href={`${guideTypeUrl}#${stopNumber}`}>
+                <TransitionLink
+                  href={`${guideTypeUrl}#${currentStop.number}`}
+                  onMouseEnter={() => {
```

Comment:
updating the `view-transition-name` on mouseenter was the only way I could find to consistently have it changed in time to work. I haven't tested this with a touch device (where we're most likely to want this to work) and I'm not convinced it will work there – open to any other ideas if you have them

Reply:
Confirmed as working on iOS and Android